airbyte-integrations/connectors/source-google-sheets/Dockerfile

```
@@ -0,0 +1,15 @@
+FROM airbyte/integration-base-python:dev
+
+RUN apt-get update && apt-get install -y jq curl bash
```

```suggestion
RUN apt-get update && apt-get install -y jq curl bash && rm -rf /var/lib/apt/lists/*
```

Can do this like other integrations to reduce the image size.